e2etest/l2tests/assignment.go

```diff
@@ -222,12 +237,18 @@ var _ = ginkgo.Describe("IP Assignment", func() {
 			resources := internalconfig.ClusterResources{
 				Pools: []metallbv1beta1.IPAddressPool{namespacePoolWithLowerPriority, namespaceLabelPoolWithHigherPriority, namespacePoolNoPriority},
 			}
-			err := ConfigUpdater.Update(resources)
+			err = ConfigUpdater.Update(resources)
 			framework.ExpectNoError(err)
 
-			svc1, _ := testservice.CreateWithBackend(cs, f.Namespace.Name, "svc-test-ns-label-pool-1")
-			svc2, _ := testservice.CreateWithBackend(cs, f.Namespace.Name, "svc-test-ns-label-pool-2")
-			svc3, _ := testservice.CreateWithBackend(cs, f.Namespace.Name, "svc-test-ns-label-pool-3")
+			updateSvcLabelsFn := func(svc *corev1.Service) {
```

Comment:
why do we need to update the labels to the service too? The selector is only against the namespace, right?

Reply:
ok, i thought you're also asking to update service with those labels to confirm no clash b/w ns and svc label selectors.

Reply:
ah no no, we just need to make the labels we are using in the ns selector stable.